tools/check_format.py

```diff
@@ -172,14 +172,64 @@ def fixJavaProtoOptions(file_path):
   return []
 
 
-def checkJavaProtoOptions(file_path):
+# Add 'option java_outclass_name = FooBarProto' for foo_bar.proto
+def fixJavaOuterClassnameProtoOption(file_path):
+  file_name = os.path.basename(file_path)[:-len(".proto")]
+  if "-" in file_name or "." in file_name or not file_name.islower():
+    return ["Unable to decide java_outer_classname for proto file: %s" % file_path]
+
+  for line in fileinput.FileInput(file_path):
+    if "option java_outer_classname = " in line:
+      return []
+
+  to_add = "option java_outer_classname = \"" + ''.join(
+      x.title() for x in file_name.split('_')) + "Proto\";\n"
+
+  for line in fileinput.FileInput(file_path, inplace=True):
+    if line.startswith("package "):
+      line = line.replace(line, line + to_add)
+    sys.stdout.write(line)
+
+  return []
+
+
+def fixJavaMultipleFilesProtoOption(file_path):
+  for line in fileinput.FileInput(file_path):
+    if "option java_multiple_files = " in line:
+      return []
+
+  for line in fileinput.FileInput(file_path, inplace=True):
+    if line.startswith("package "):
+      line = line.replace(line, line + "option java_multiple_files = true;\n")
+    sys.stdout.write(line)
+
+  return []
+
+
+def checkJavaPackageProtoOption(file_path):
   for line in fileinput.FileInput(file_path):
     if "option java_package = \"io.envoyproxy.envoy" in line:
       return []
 
   return ["Java proto option 'java_package' not set correctly for file: %s" % file_path]
 
 
+def checkJavaOuterClassnameProtoOption(file_path):
+  for line in fileinput.FileInput(file_path):
+    if "option java_outer_classname = " in line:
+      return []
+
+  return ["Java proto option 'java_outer_classname' not set for file: %s" % file_path]
+
+
+def checkJavaMultipleFilesProtoOption(file_path):
+  for line in fileinput.FileInput(file_path):
+    if "option java_multiple_files = " in line:
```

Comment:
Maybe check that this is set to true? Or are there cases where you wouldn't want that?

Reply:
This only checks if the developer forgot to set it. If the developer sets it to false, that is unusual, might imply that they purposely want it to be false.

Reply:
Ok, makes sense